#26 added additional views for agenda page

khanabdu25 opened the pull request:
- agenda page now has dayview, timeline view, and schedule view
- made subdir for pages - agenda_pages

khanabdu25 committed:
#26 added additional views for agenda page

- agenda page now has dayview, timeline view, and schedule view
- made subdir for pages - agenda_pages

shouryan01 commented:
amazing work @khanabdu25 ! Looks really good.

@rnair98 we'll merge this AFTER the meeting on Thu

khanabdu25 commented:
> amazing work @khanabdu25 ! Looks really good.
> 
> @rnair98 we'll merge this AFTER the meeting on Thu

Thank youww
I'm still working on the popup on tap though, will hopefully have it ready by dev's meeting tmrw

khanabdu25 committed:
added expanded view on tap

clicking on an event in any of the views now opens a details page

khanabdu25 committed:
#26 added details page on event calendar tap

details page functionality on tap of an event from any of the views is now present
